README.md

```diff
@@ -153,6 +153,7 @@ More information on contributing and the general code of conduct for discussion
 | Word to PDF                              | [Word to PDF](https://github.com/DhanushNehru/Python-Scripts/tree/master/Word%20to%20PDF%20converter)                                                    | A Python script to convert an MS Word file to a PDF file.                                                                                                         |
 | Youtube Downloader                       | [Youtube Downloader](https://github.com/DhanushNehru/Python-Scripts/tree/master/Youtube%20Downloader)                                                    | Downloads any video from [YouTube](https://youtube.com) in video or audio format!                                                                                 |
 | Youtube Playlist Info Scraper            | [Youtube Playlist Info Scraper](https://github.com/DhanushNehru/Python-Scripts/tree/master/Youtube%20Playlist%20Info%20Scraper)                          | This python module retrieve information about a YouTube playlist in json format using playlist link.                                                              |
+| Mood based youtube song generator        | [Mood based youtube song generator](https://github.com/DhanushNehru/Python-Scripts/tree/master/Mood%20based%20youtube%20song%20generator)                | This Python script fetches a random song from YouTube based on your mood input and opens it in your default web browser.                                                              |
```

Comment:
Thank you for fixing that. However, I still have an issue. The data is not in alphabetical order. In our README, we mention that the table data should be organized in alphabetical order. Could you please consider placing your script after "M" in the list to maintain alphabetical order?

Reply:
Sure thing, Fixed.
Thank you for the review